installation-and-deployment/deploying-with-helm.md

```diff
@@ -0,0 +1,59 @@
+# Deploying on Kubernetes with Helm Chart
+
+To deploy Aqueduct on a Kubernetes cluster, you can use the helm chart we've created. Please follow the steps below.
+
+### Prerequisites
+
+* Follow the instruction [here](https://helm.sh/docs/intro/install/) to install Helm on your local machine.
+* You need to have access to an existing Kubernetes cluster. This means your local machine needs to have a kubeconfig file (typically located in `~/.kube/config`) that points to your cluster.
```

Comment:
this prereq should come first

"that points to the cluster you want to run Aqueduct on"